modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -111,30 +110,53 @@ IMPORTANT NOTES:
 
 == Create Let’s Encrypt’s Config Files
 
-* Create the following files in the Let’s Encrypt directory. They will help to maintain your certificates.
-* Replace the path to Certbot and the Certbot script name based on your installation. +
-You can find it by running `which certbot`.
-* Rename <your-domain-name>.sh with the name of the domain(s) you want to issue a certificate for. +
-For example, the script could be renamed to `your-domain-name.com.sh`.
-* Make all files executable except `cli.ini` by running `sudo chmod +x <script-name>`.
-
-NOTE: All scripts have to be executed with `sudo`.
+Because remembering all the possible options for certbot, the following scripts ease the use for common tasks because of their self speaking name.
```

Comment:
```suggestion
Because remembering all the possible options for certbot is difficult, the following scripts ease the use for common tasks because of their self-descriptive name.
```